cfme/tests/cli/test_appliance_console.py

```diff
@@ -70,6 +72,13 @@ def secondary_dns(temp_appliance_preconfig_long):
     return secondary_dns
 
 
+@pytest.fixture
+def freeipa_provider():
```

Comment:
I don't think this should be hardcoded, the existing fixture should be used and `prov_key` should be parametrized, ideally from the yaml auth keys that are available.

https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/authentication.py#L99

Reply:
**Optional**